news.html

```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">
              href="https://docs.dockstore.org/en/stable/getting-started/getting-started-with-notebooks.html">
               Notebooks</a>
       </h5>
-        <span>Learn how to register notebooks in Dockstore</span>
+        <span>Learn how to register notebooks in Dockstore. For a simple notebook example, explore this <a target="_blank" rel="noopener noreferrer"
+          href="https://dockstore.org/notebooks/github.com/dockstore/notebook-example/running-code:main?tab=info">
```

Comment:
Is this flipped?
Working link is `example-notebook`

Reply:
Good catch, updated :slightly_smiling_face: